tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -349,14 +349,26 @@ async function checkTrace(traceId, expectedTraceProcessSequence) {
 
   // searching through the trace-graph for the expected span sequence staring at the root element
   const wasFound = await findSpanSequence(expectedTraceProcessSequence, 0, traceDAG[0], traceData);
+  if (!wasFound) {
+    debug(`not all expected spans found in the expected order:`);
+    for (let i = 0; i < expectedTraceProcessSequence.length; i++) {
+      debug(`${expectedTraceProcessSequence[i]}`);
+    }
+  } else {
+    debug(`all expected spans found in the expected order.`);
+  }
   expect(wasFound).to.be.true;
 }
 
 // findSpanSequence recursively searches through the trace-graph to find all expected spans in the right, consecutive
 // order while ignoring the spans that are not expected.
 async function findSpanSequence(expectedSpans, pos, currentSpan, traceData) {
+  const actualSpan = traceData.processes[currentSpan.processID].serviceName;
+  const expectedSpan = expectedSpans[pos];
+  const match = (actualSpan === expectedSpan);
+  debug(`${match ? '✓' : '𐄂'}${await buildLevel(pos)} ${actualSpan} ${match ? '' : `expected ${expectedSpan}`}`);
```

Comment:
(nit)
I am also not sure about the `✓` and `x`, I think the tests will display the same check mark if they are green and will throw an assert error if they fail, so you won't need the `x`. IMO you could remove this characters.